specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4009,8 +4012,9 @@
         "tags": [
           "Tags"
         ],
-        "operationId": "Tags_ResourceCreate",
-        "description": "Create or Replace existing tags with passing in tags.",
+        "operationId": "Tags_CreateOrUpdateTagsOnObject",
```

Comment:
Format should be Noun_Verb. Suggest TagsOnObject_CreateOrUpdate

Reply:
This will confuse the users with inconsistencies between the API doc and documents concept of Azure tags. The existing "Tags_*" operations should have been named differently, but since the renaming is not possible now, we are making this change to keep things a little clearer for the partners.